### ISSUE TITLE
Support IPv6 ?

### ISSUE THREAD

**Issue by hotarugali** [chubin/wttr.in#598]
When I try `wttr.in/<IPv6addr>`，it cannot fetch correct info. So does `wttr.in` support IPv6 ? If does how to fetch correct info ?
Thanks.

**Comment by chubin** [comment]
No, it does not (see also #20), but the support should be added
